### ISSUE TITLE
How to leverage SCAD for training with multiple source datasets while effectively utilizing labeled target data.

### ISSUE THREAD

**Issue by mingjiang7940** [CompBioT/SCAD#4]
Hi,

Thank you for providing a nice tool. I would like to use SCAD to train with multiple source datasets (for example, various cancer datasets) along with a labeled target training dataset. How should I approach this?

From my understanding, SCAD allows only a single source matrix input during model training and does not use the target training data labels for the training process (they are only used for evaluation).

In summary, how can I train the model using multiple source datasets and make use of the valuable target data labels? I look forward to your guidance. Thank you!   @CompBioT

**Comment by TimH94** [comment]
Hi mingjiang, thank you for your interest. 

I think it depends on what you want to use those valuable target data labels. 

If you just want to use them as part of the training set, you can try to consider labelled single cell data as 'source domain' data (adding them to source domain drug-specific GDSC training set), since SCAD feature extractor is proposed to capture similar feature representations for bulk and scRNA-seq data.

**Comment by CompBioT** [comment]
Hi mingjiang, thank you for your interest and sorry for the late reply.

If you just want to use them as part of the training set, you can try to consider labelled single cell data as 'source domain' data (adding them to source domain drug-specific GDSC training set), since SCAD feature extractor is proposed to capture similar feature representations for bulk and scRNA-seq data.

**Comment by mingjiang7940** [comment]
I see.
Thanks for your reply!
